unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -551,5 +551,16 @@ public static bool TryFindChildRecursively(this Transform transform, string name
             }
             return false;
         }
+
+        public static bool IsPointerOverUIElement(Vector3 mousePosition)
+        {
+            var eventData = new PointerEventData(EventSystem.current);
+            eventData.position = mousePosition;
+            var results = new List<RaycastResult>();
+            EventSystem.current.RaycastAll(eventData, results);
+            return results.Count > 1;
```

Comment:
Maybe you already tried but I'd like to know, could we avoid the raycast and list creation with any of the following checks? (Maybe they don't work in this case)

`EventSystem.current.currentSelectedGameObject != null`
or
`EventSystem.current.alreadySelecting`
or
`EventSystem.current.IsPointerOverGameObject()`